myTrip/trip/models.py

```diff
@@ -112,3 +118,17 @@ def delete_by_id(trip_id):
         trip = Trip.objects.get(id=trip_id)
         trip.delete()
         return None
+
+    def get_trips(user_id,page=1,step=5):
+        """
+        Returns the last 5 trips by the user
```

Comment:
if you only need fixed '5' you should delete it from arguments
and replace word 'step' below into int '5'

or you could do a function get_trips, with which user can define himself how mych trips he needs back, and just set step=5 by default.